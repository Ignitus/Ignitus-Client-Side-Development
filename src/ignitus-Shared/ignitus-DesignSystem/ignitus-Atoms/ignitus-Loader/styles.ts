import styled from '@emotion/styled';
import { IgnitusLogo } from '../../ignitus-Assets/ignitus-Icons/ignitusLogo';
import { IgnitusBlue, GreyLight } from '../colors';

export const Spinner = styled.div`
  position: absolute;
  top: 0;
  left: 0;

  margin: auto;
  border: 1rem solid ${GreyLight};
  border-radius: 50%;
  border-top: 1rem solid ${IgnitusBlue};
  height: calc(100% - 2rem);
  width: calc(100% - 2rem);
  -webkit-animation: spin 2.5s linear infinite;
  animation: spin 3s linear infinite;

  @keyframes spin {
    0% {
      transform: rotate(0deg);
    }
    100% {
      transform: rotate(360deg);
    }
  }
`;

export const Logo = styled(IgnitusLogo)`
  height: 65%;
  fill: ${IgnitusBlue};
`;

export const Container = styled.div`
  position: relative;
  display: flex;
  justify-content: center;
  align-items: center;
  margin: auto;
  width: 10rem;
  height: 10rem;
`;
