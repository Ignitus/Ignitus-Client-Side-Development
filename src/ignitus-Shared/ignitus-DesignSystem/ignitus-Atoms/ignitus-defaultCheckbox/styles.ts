import styled from '@emotion/styled';
import { Paragraph } from '../typography';
import { LabelType } from './types';

export const CheckBox = styled.input`
  height: 1rem;
  width: 1rem;
  outline: none;
`;
export const Label = styled.label`
  margin-left: 0.4rem;
`;

export const StyledParagraph = styled(Paragraph)<LabelType>`
  font-size: ${props => props.size};
`;

export const Container = styled.div`
  display: inline-flex;
  align-items: center;
`;
