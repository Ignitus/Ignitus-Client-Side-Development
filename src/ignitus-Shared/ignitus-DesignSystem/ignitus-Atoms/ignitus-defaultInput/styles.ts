import styled from '@emotion/styled';
import * as C from '../colors';
import * as F from '../fonts';

export const Input = styled.input`
  height: ${props => props.height};
  width: ${props => props.width};
  background-color: ${C.GreyLight};
  padding-left: 2rem;
  color: ${C.GreySecondaryText};
  font-weight: ${F.SemiBold};
  font-size: ${F.XL};
  border: none;
  border-radius: 17px;
  outline: none;
  margin: 0.3rem 0;

  ::placeholder {
    color: ${C.GreySecondaryText};
    font-weight: ${F.SemiBold};
    font-size: ${F.MD};
    opacity: 0.5;
  }
`;
