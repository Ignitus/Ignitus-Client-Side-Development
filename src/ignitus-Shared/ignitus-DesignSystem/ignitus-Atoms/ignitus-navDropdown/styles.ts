import styled from '@emotion/styled/macro';
import { White, IgnitusBlue } from '../colors';
import { Icon as I } from '../../../ignitus-Utilities/Components/icon';
import { StyleProps } from './types';
import { MD } from '../fonts';

export const Container = styled.div`
  display: flex;
  flex-direction: column;
  width: 12rem;
`;

export const Column = styled.div`
  background-color: ${White};
  border-radius: 1rem;
  margin-top: 1rem;
`;

export const NavigationHeading = styled.div`
  align-items: center;
  color: ${IgnitusBlue};
  cursor: pointer;
  display: flex;
  font-size: ${MD};
  justify-content: flex-end;
`;

export const Icon = styled(I)<StyleProps>`
  height: 2rem;
  fill: ${IgnitusBlue};
  margin: 0 0.5rem;
`;

export const Text = styled.div`
  text-align: center;
`;
