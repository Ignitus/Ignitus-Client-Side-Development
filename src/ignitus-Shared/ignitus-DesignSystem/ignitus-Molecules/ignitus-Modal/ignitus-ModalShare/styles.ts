import styled from '@emotion/styled';
import { Paragraph as P, Normal, MD, White, boxShadowColor } from '../../../..';
import { Icon } from '../../../../ignitus-Utilities/Components/icon';

import { flexibleColDiv, flexibleRowDiv } from '../../../shared';

export const ModalShare = styled.div`
  background-color: ${White};
  background-size: contain;
  border-radius: 1rem;
  display: inline-flex;
  flex-direction: column;
  box-shadow: 0 2px 4px 0 ${boxShadowColor};
  width: 35rem;
`;

export const TopSection = styled(flexibleRowDiv)`
  padding: 1.5rem;
  justify-content: space-between;
`;

export const BottomSection = styled(flexibleColDiv)`
  padding: 0.4rem 0.8rem 1.5rem 1.5rem;
  align-items: flex-start;
`;

export const IconsContainer = styled(flexibleRowDiv)`
  border-radius: 1rem;
  display: inline-flex;
`;

export const IconContainer = styled.button`
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: ${props => props.color};
  outline-color: transparent;
`;

export const Paragraph = styled(P)`
  font-weight: ${Normal};
  font-size: ${MD};
  margin: 2rem 0.6rem 0.6rem 0.6rem;
`;

export const StyledIcon = styled(Icon)`
  width: 1.4rem;
  fill: ${props => props.color};
  transition: 0.3s;
  flex: 1;
  margin-top: 0.2rem;
`;
