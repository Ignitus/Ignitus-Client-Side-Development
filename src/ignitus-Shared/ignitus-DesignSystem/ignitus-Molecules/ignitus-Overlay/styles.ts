import styled from '@emotion/styled';
import { flexibleColDiv } from '../../shared';
import { Icon } from '../../../ignitus-Utilities/Components/icon';
import { Heading1 } from '../../..';

export const Overlay = styled.div`
  position: relative;
  display: flex;
  align-items: center;
  justify-content: center;
  background-color: rgba(0, 0, 102, 0.89);
  backdrop-filter: blur(10px);
  width: 100%;
  height: 100%;
`;

export const Logo = styled(Icon)`
  margin-bottom: 4rem;
  height: 7.5rem;
  path {
    fill: white;
  }
`;

export const Heading = styled(Heading1)`
  margin-bottom: 1.5rem;
`;

export const MessageSection = flexibleColDiv;
