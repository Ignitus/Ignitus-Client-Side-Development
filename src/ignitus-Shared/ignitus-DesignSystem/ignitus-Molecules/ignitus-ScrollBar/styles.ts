import styled from '@emotion/styled';

export const Scroll = styled.div`
  margin: 30px;
  float: left;
  height: 300px;
  width: 65px;
  background: #f5f5f5;
  overflow-y: scroll;
  margin-bottom: 25px;

  ::-webkit-scrollbar {
    width: 12px;
    background: #f5f5f5;
  }

  ::-webkit-scrollbar-track {
    box-shadow: inset 0 0 6px rgba(0, 0, 0, 0.3);
    border-radius: 10px;
    background-color: #f5f5f5;
  }

  ::-webkit-scrollbar-thumb {
    border-radius: 10px;
    background-color: #f90;
  }
`;

export const Overflow = styled.div`
  min-height: 450px;
`;
