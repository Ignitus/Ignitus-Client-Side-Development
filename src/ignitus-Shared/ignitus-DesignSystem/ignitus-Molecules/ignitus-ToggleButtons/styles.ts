import styled from '@emotion/styled';

import { Icon } from '../../../ignitus-Utilities/Components/icon';
import * as B from '../../ignitus-Atoms/buttons';

export const ButtonLeft = styled(B.Button)`
  font-weight: 600;
  margin: 0;
  border: 0;
  float: left;
  border-radius: 20px 0 0 20px;
`;

export const ButtonRight = styled(B.Button)`
  font-weight: 600;
  margin: 0;
  border: 0;
  float: left;
  border-radius: 0 20px 20px 0;
`;

export const ToggleButtonContainer = styled.span`
  cursor: pointer;
`;

export const StyledIcon = styled(Icon)`
  width: 1.4rem;
  fill: ${props => props.color};
  transition: 0.3s;
  flex: 1;
  margin: 0.03rem;
`;
