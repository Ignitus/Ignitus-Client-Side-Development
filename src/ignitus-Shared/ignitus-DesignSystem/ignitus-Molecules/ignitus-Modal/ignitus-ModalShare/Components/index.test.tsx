import React from 'react';
import { mount, render } from 'enzyme';
import emotionSerializer, { matchers } from 'jest-emotion';
import { ModalShare } from './index';
import { Heading1, DefaultIconInput } from '../../../../..';
import * as O from '../styles';
import { GreyOne } from '../../../../ignitus-Atoms/colors';
import * as C from '../../../../ignitus-Atoms/colors';

const props = {
  title: 'title',
  Share: '',
  Link: '',
  LinkedinColor: C.LinkedinIconColor,
  TwitterColor: C.TwitterIconColor,
  RedditColor: C.RedditIconColor,
  FacebookColor: C.FacebookIconColor,
  White: C.White,
  GreyOne: C.GreyOne,
};

let wrapper;

expect.addSnapshotSerializer(emotionSerializer);
expect.extend(matchers);

describe('<ModalShare />', () => {
  beforeEach(() => {
    wrapper = mount(<ModalShare {...props} />);
  });

  it('should render', () => {
    wrapper = render(<ModalShare {...props} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should render StyledIcon, DefaultIconInput', () => {
    expect(wrapper.find(O.StyledIcon).length).toBe(5);
    expect(wrapper.find(DefaultIconInput).length).toBe(1);
  });

  it('should render colors of StyledIcon, IconsContainer as passed in props', () => {
    expect(
      wrapper
        .find(O.StyledIcon)
        .at(0)
        .prop('color'),
    ).toBe(GreyOne);
    expect(
      wrapper
        .find(O.IconContainer)
        .at(0)
        .prop('color'),
    ).toBe(props.White);
    expect(
      wrapper
        .find(O.IconContainer)
        .at(1)
        .prop('color'),
    ).toBe(props.FacebookColor);
    expect(
      wrapper
        .find(O.IconContainer)
        .at(2)
        .prop('color'),
    ).toBe(props.RedditColor);
    expect(
      wrapper
        .find(O.IconContainer)
        .at(3)
        .prop('color'),
    ).toBe(props.TwitterColor);
    expect(
      wrapper
        .find(O.IconContainer)
        .at(4)
        .prop('color'),
    ).toBe(props.LinkedinColor);
  });

  it('should render text as passed in props', () => {
    expect(wrapper.find(Heading1).text()).toBe(props.title);
    expect(
      wrapper
        .find(O.Paragraph)
        .at(0)
        .text(),
    ).toBe(props.Link);
    expect(
      wrapper
        .find(O.Paragraph)
        .at(1)
        .text(),
    ).toBe(props.Share);
  });
});
