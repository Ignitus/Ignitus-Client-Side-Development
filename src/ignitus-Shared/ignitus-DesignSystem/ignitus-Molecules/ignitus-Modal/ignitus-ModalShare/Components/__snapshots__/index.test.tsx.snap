// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ModalShare /> should render 1`] = `
.emotion-19 {
  background-color: #ffffff;
  background-size: contain;
  border-radius: 1rem;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  box-shadow: 0 2px 4px 0 rgba(0,0,0,0.05);
  width: 35rem;
}

.emotion-3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 1.5rem;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.emotion-0 {
  font-weight: 500;
  font-size: 32px;
  color: #000066;
}

.emotion-2 {
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: #ffffff;
  outline-color: transparent;
}

.emotion-1 {
  width: 1.4rem;
  fill: #9C9C9C;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-top: 0.2rem;
}

.emotion-18 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 0.4rem 0.8rem 1.5rem 1.5rem;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
}

.emotion-4 {
  font-size: 16px;
  font-weight: 400;
  color: #2D2D2D;
  margin: 0;
  font-weight: 400;
  font-size: 18px;
  margin: 2rem 0.6rem 0.6rem 0.6rem;
}

.emotion-7 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  border-radius: 0.5rem;
  padding: 0.7rem;
  background: #E0E0E0;
  border: 1px solid #E0E0E0;
}

.emotion-7:hover {
  border: 1px solid #000066;
}

.emotion-5 {
  outline: none;
  border: none;
  background: transparent;
  font-size: 16px;
  font-weight: 400;
  color: #2D2D2D;
}

.emotion-6 {
  height: 1.4rem;
  width: 1.4rem;
  cursor: pointer;
  fill: #000066;
}

.emotion-17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 1rem;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
}

.emotion-10 {
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: #3b5998;
  outline-color: transparent;
}

.emotion-9 {
  width: 1.4rem;
  fill: #ffffff;
  -webkit-transition: 0.3s;
  transition: 0.3s;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  margin-top: 0.2rem;
}

.emotion-12 {
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: #FF4301;
  outline-color: transparent;
}

.emotion-14 {
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: #00acee;
  outline-color: transparent;
}

.emotion-16 {
  border: none;
  cursor: pointer;
  overflow: hidden;
  margin: 0 0.2rem;
  height: 2.8rem;
  width: 2.8rem;
  border-radius: 50%;
  background-color: #0e76a8;
  outline-color: transparent;
}

<div
  class="emotion-19"
>
  <div
    class="emotion-3"
  >
    <h1
      class="emotion-0"
    >
      title
    </h1>
    <button
      class="emotion-2"
      color="#ffffff"
    >
      <svg
        aria-label="CrossIcon"
        class="emotion-1"
        color="#9C9C9C"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          CrossIcon
        </title>
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M18.3 5.71c-.39-.39-1.02-.39-1.41 0L12 10.59 7.11 5.7c-.39-.39-1.02-.39-1.41 0-.39.39-.39 1.02 0 1.41L10.59 12 5.7 16.89c-.39.39-.39 1.02 0 1.41.39.39 1.02.39 1.41 0L12 13.41l4.89 4.89c.39.39 1.02.39 1.41 0 .39-.39.39-1.02 0-1.41L13.41 12l4.89-4.89c.38-.38.38-1.02 0-1.4z"
        />
      </svg>
    </button>
  </div>
  <div
    class="emotion-18"
  >
    <p
      class="emotion-4"
    />
    <div
      class="emotion-7"
    >
      <input
        class="emotion-5"
        placeholder="Paste URL Here"
        type="string"
        value=""
      />
      <svg
        aria-label="Copy Icon"
        class="emotion-6"
        viewBox="0 0 24 24"
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Copy Icon
        </title>
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M15 1H4c-1.1 0-2 .9-2 2v13c0 .55.45 1 1 1s1-.45 1-1V4c0-.55.45-1 1-1h10c.55 0 1-.45 1-1s-.45-1-1-1zm.59 4.59l4.83 4.83c.37.37.58.88.58 1.41V21c0 1.1-.9 2-2 2H7.99C6.89 23 6 22.1 6 21l.01-14c0-1.1.89-2 1.99-2h6.17c.53 0 1.04.21 1.42.59zM15 12h4.5L14 6.5V11c0 .55.45 1 1 1z"
        />
      </svg>
    </div>
    <p
      class="emotion-4"
    />
    <div
      class="emotion-17"
    >
      <button
        class="emotion-10"
        color="#3b5998"
      >
        <svg
          aria-label="Facebook Icon"
          class="emotion-9"
          color="#ffffff"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            Facebook Icon
          </title>
          <path
            d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
          />
        </svg>
      </button>
      <button
        class="emotion-12"
        color="#FF4301"
      >
        <svg
          aria-label="Reddit Icon"
          class="emotion-9"
          color="#ffffff"
          enable-background="new 0 0 24 24"
          viewBox="0 0 24 24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            Reddit Icon
          </title>
          <path
            d="m21.325 9.308c-.758 0-1.425.319-1.916.816-1.805-1.268-4.239-2.084-6.936-2.171l1.401-6.406 4.461 1.016c0 1.108.89 2.013 1.982 2.013 1.113 0 2.008-.929 2.008-2.038s-.889-2.038-2.007-2.038c-.779 0-1.451.477-1.786 1.129l-4.927-1.108c-.248-.067-.491.113-.557.365l-1.538 7.062c-2.676.113-5.084.928-6.895 2.197-.491-.518-1.184-.837-1.942-.837-2.812 0-3.733 3.829-1.158 5.138-.091.405-.132.837-.132 1.268 0 4.301 4.775 7.786 10.638 7.786 5.888 0 10.663-3.485 10.663-7.786 0-.431-.045-.883-.156-1.289 2.523-1.314 1.594-5.115-1.203-5.117zm-15.724 5.41c0-1.129.89-2.038 2.008-2.038 1.092 0 1.983.903 1.983 2.038 0 1.109-.89 2.013-1.983 2.013-1.113.005-2.008-.904-2.008-2.013zm10.839 4.798c-1.841 1.868-7.036 1.868-8.878 0-.203-.18-.203-.498 0-.703.177-.18.491-.18.668 0 1.406 1.463 6.07 1.488 7.537 0 .177-.18.491-.18.668 0 .207.206.207.524.005.703zm-.041-2.781c-1.092 0-1.982-.903-1.982-2.011 0-1.129.89-2.038 1.982-2.038 1.113 0 2.008.903 2.008 2.038-.005 1.103-.895 2.011-2.008 2.011z"
          />
        </svg>
      </button>
      <button
        class="emotion-14"
        color="#00acee"
      >
        <svg
          aria-label="Twitter Icon"
          class="emotion-9"
          color="#ffffff"
          fill="#00acee"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            Twitter Icon
          </title>
          <path
            d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
          />
        </svg>
      </button>
      <button
        class="emotion-16"
        color="#0e76a8"
      >
        <svg
          aria-label="LinkedIn Icon"
          class="emotion-9"
          color="#ffffff"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <title>
            LinkedIn Icon
          </title>
          <path
            d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
          />
        </svg>
      </button>
    </div>
  </div>
</div>
`;
